core/lib/spree/permitted_attributes.rb

```diff
@@ -31,7 +31,7 @@ module PermittedAttributes
 
     @@address_attributes = [
       :id, :firstname, :lastname, :first_name, :last_name,
-      :address1, :address2, :city, :country_id, :state_id,
+      :address1, :address2, :city, :country_iso, :state_id,
```

Comment:
@przemosk we need to leave both `country_id` and `country_iso` so all old endpoints will work properly